afisha-client/src/app/shared/services/event.service.ts

```diff
@@ -0,0 +1,51 @@
+import {Injectable} from '@angular/core';
+import {IEvent} from "../models/event.model";
+import {HttpClient, HttpHeaders} from "@angular/common/http";
+import {Observable} from "rxjs/Observable";
+import {PATH} from "../constants/path.constant";
+
+
+@Injectable()
+export class EventService {
+
+  constructor(private http: HttpClient) {
+
+  }
+
+  getEvents(): Observable<IEvent[]>{
+    return this.http.get<IEvent[]>(PATH.API + 'events', {
+      headers: new HttpHeaders({
```

Comment:
Firstly - If you use something that repeated in every request - this thing have to defined in interceptor.
Secondly - in your case header is redundant. Angular already sent all requests as 'application/json'